devops/nginx/troubleshooting/ssl.md

```diff
@@ -0,0 +1,29 @@
+## SSL
+
+### Enabling support for older devices & browsers
```

Comment:
It's probably worth mentioning that `ssl_prefer_server_ciphers` must be set to `on` for `ssl_ciphers` to work correctly:
```
ssl_prefer_server_ciphers on;
```